site/en_AU.all.json

```diff
@@ -352,5 +356,81 @@
+  "all_rights_reserved": { "other": "All rights reserved. No part of this site may be reproduced without our written permission." },
+
+  "users_gender_none": { "other": "Not specified" },
+  "users_gender_female": { "other": "Female" },
+  "users_gender_male": { "other": "Male" },
+  "users_gender_diverse": { "other": "Diverse" },
+  "users_gender_other": { "other": "Other" },
+
+  "plan_label_owned": { "other": "" },
+
+  "pricing_quality_hd": { "other": "TODO" },
+  "pricing_quality_sd": { "other": "TODO" },
+  "pricing_quality_hd_only": { "other": "TODO" },
+  "pricing_quality_sd_only": { "other": "TODO" },
+
+  "shopping_card_save_card": { "other": "Save this card for future use" },
+  "shopping_card_button_change": { "other": "Show all saved cards" },
+  "shopping_card_existing_description": { "other": "{{.Card.Brand}} ending in {{.Card.Number}}" },
+  "shopping_card_existing_expiry": { "other": "(expires {{.Card.Expiry}})" },
+  "shopping_card_existing_expired": { "other": "(expired)" },
+  "shopping_card_use_other": { "other": "Use a new card" },
+  "shopping_card_use_new_card": { "other": "Change card" },
+
+  "shopping_info_ownership_plan": { "other": "TODO" },
+
+  "shopping_price_title_plan": { "other": "TODO" },
+  "shopping_price_plan_note": { "other": "TODO" },
+  "shopping_price_plan_note_interval": { "other": "TODO" },
+
+  "shopping_enter_card_prompt_plan": { "other": "TODO" },
+
+  "shopping_action_credit": { "other": "TODO" },
+  "shopping_action_plan": { "other": "TODO" },
+
+  "shopping_complete_subscription": { "other": "TODO" },
+  "shopping_complete_subscription_browse": { "other": "TODO" },
+
+  "shopping_info_rental_period_coming_soon": { "other": "from the date and time of release." },
+  "shopping_info_plan_offer": { "other": "TODO" },
+  "shopping_info_trial_period_offer": { "other": "TODO" },
+
+  "usersubscriptions_change_payment_method_change": { "other": "TODO" },
+  "usersubscriptions_change_payment_method_modal_title": { "other": "TODO" },
+  "usersubscriptions_change_payment_method_modal_submit": { "other": "TODO" },
+
+  "usersubscriptions_subscription_expiry": { "other": "TODO" },
+  "usersubscriptions_subscription_status_cancelled": { "other": "TODO" },
+  "usersubscriptions_subscription_status_errored": { "other": "TODO" },
+  "usersubscriptions_subscription_status_past_due": { "other": "TODO" },
+  "usersubscriptions_subscription_status_trialing": { "other": "TODO" },
+  "usersubscriptions_subscription_unsubscribe": { "other": "TODO" },
+  "usersubscriptions_unsubscribe_modal_cancel": { "other": "TODO" },
+  "usersubscriptions_unsubscribe_modal_confirm": { "other": "TODO" },
+  "usersubscriptions_unsubscribe_modal_title": { "other": "TODO" },
+  "usersubscriptions_unsubscribe_modal_body": { "other": "TODO" },
+
+  "shopping_card_update_reason_none": { "other": "TODO" },
+  "shopping_card_update_reason_ + this.updateCardReason": { "other": "TODO" },
```

Comment:
Looks like the various `updateCardReason`'s need to be enumerated

Reply:
I'm going to remove that one... updateCardReason comes from a query string and I can't find an example of it being used anywhere.

Reply:
Sweet